test/Grpc.AspNetCore.Server.Tests/TestObjects/TestEndpointRouteBuilder.cs

```diff
@@ -25,7 +25,7 @@ namespace Grpc.AspNetCore.Server.Tests.TestObjects
 {
     public class TestEndpointRouteBuilder : IEndpointRouteBuilder
     {
-        public IServiceProvider ServiceProvider { get; set; }
+        public IServiceProvider? ServiceProvider { get; set; }
```

Comment:
Based on usage, it seems like we would also like for this property to never be null either. Use a constructor?

Reply:
Sure. It doesn't really matter either way